ocean-client/src/programs/vault-maxi-program.ts

```diff
@@ -179,6 +224,12 @@ export class VaultMaxiProgram extends CommonProgram {
         const neededrepay = BigNumber.max(
                     new BigNumber(vault.loanValue).minus( new BigNumber(vault.collateralValue).dividedBy(this.targetCollateral)),
                     nextLoanValue(vault).minus(nextCollateralValue(vault).div(this.targetCollateral)))
+        if(neededrepay.lte(0)) {
+            console.error("negative repay, whats happening? loans:"+ vault.loanValue+ "/"+ nextLoanValue(vault)
+            +" cols:"+ vault.collateralValue+"/"+nextCollateralValue(vault)+" target:"+this.targetCollateral)
```

Comment:
shouldn't we multiply targetCollateral by 100 in order to correctly visualize it?

Reply:
nvm this is just the error log

Reply:
I also added the target to the status notification. will multiply there